### PR TITLE
[FIX] middleware/testRunner: Update resources from OpenUI5

### PR DESCRIPTION
**Pull request by RandomByte**

### CODE REVIEW

**Comment by RandomByte** [comment]
Not yet tested

**Comment by matz3** [comment]
Added a script to do the update. But I also didn't verify the changes at runtime, yet.

File: scripts/update-testRunner-resources.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+curl -L -O https://github.com/SAP/openui5/raw/master/src/sap.ui.core/test/sap/ui/qunit/TestRunner.js
+curl -L -O https://github.com/SAP/openui5/raw/master/src/sap.ui.core/test/sap/ui/qunit/testrunner.css
+curl -L -O https://github.com/SAP/openui5/raw/master/src/sap.ui.core/test/sap/ui/qunit/testrunner.html
+
+# Check if there are updates
+git diff --quiet && echo "No updates. Exiting..." && exit 0
+
+# Get latest commit url for src/sap.ui.core/test/sap/ui/qunit path
+COMMIT_URL=$(curl "https://api.github.com/repos/SAP/openui5/commits?sha=master&path=src/sap.ui.core/test/sap/ui/qunit" | \
+node -r fs -e "process.stdout.write(JSON.parse(fs.readFileSync(\"/dev/stdin\", \"utf8\"))[0].html_url)")
```
Comment (RandomByte) [inline scripts/update-testRunner-resources.sh:19]: An alternative would be [jq](https://stedolan.github.io/jq/). Needs to be installed however

**Comment by matz3** [comment]
Works fine with the openui5-sample-app 👍🏻
At least when clearing the cache or after the old resources are invalidated (180s).

Why not also using ETag caching here?

**Comment by matz3** [comment]
Just pushed my change for ETag caching. We can still decide if we want to enable it or stick with the 180s caching.

**Comment by matz3** [comment]
Extracted etag caching to https://github.com/SAP/ui5-server/pull/354 so that we can just merge the update.

**Review by matz3 — APPROVED** [review]

**Pull request merged**